### PR TITLE
[WIP] Email config php

### LINKED ISSUE #19: Configurable config.php values (also after inital install)
I wanna make config.php values configurable: 

A sample config.php can look like this:

``` php
<?php
$CONFIG = array (
  'theme' => 'test',
  'trusted_domains' =>
  array (
    0 => 'owncloud.test.de',
  ),
  'apps_paths' => 
  array (
    0 => 
    array (
      'path' => '/var/www/virtual/xxx.portknox.net/htdocs/apps',
      'url' => '/apps',
      'writable' => false,
    ),
    1 => 
    array (
      'path' => '/var/www/virtual/xxx.portknox.net/htdocs/apps2',
      'url' => '/apps2',
    ),
  ),

);
```

The theme option is easy, should be like this:

``` yml
  tasks:

    - name: switch theme
      lineinfile:
        dest=/var/www/owncloud/config/config.php
        regexp='theme'
        line=" 'theme' => {{ theme }},"
```

But how can i manage `trusted_domains` and `apps_paths`. They are more complex and multiline.  

Somewhere a example or can this be a way: http://stackoverflow.com/a/24345892/4046794
@drybjed @scibi @ypid

### PR DESCRIPTION
**Pull request by Gomez**
Will fix #19 
Need further testing.

### CODE REVIEW

**Comment by Gomez** [comment]
@ypid If you find time, you can review this PR (just the code and how it is done). Needs testing & verification.

File: defaults/main.yml [hunk 1]
```diff
@@ -229,7 +229,9 @@ owncloud_upload_size: '128M'
 # .. envvar:: owncloud_php5_output_buffering
 #
 # Output buffering set in php5, with amount set in megabytes
-owncloud_php5_output_buffering: '128'
+# Setting according to: 
+# https://doc.owncloud.org/server/8.0/admin_manual/configuration_files/big_file_upload_configuration.html#configuring-php
+owncloud_php5_output_buffering: '0'
```
Comment (ypid) [inline defaults/main.yml:234]: Revert, see #21

**Comment by ypid** [comment]
I quickly check your PR. Please use the trick with JSON decode: https://github.com/debops/ansible-owncloud/blob/e1a06ba27a9a3d84378e0c2255f08949165f49d9/templates/srv/www/sites/config/config.default.php.j2#L12

Also, to properly close #19 I would recommend to allow more flexibility like additionally adding a `owncloud_custom_conf_map: {}` for custom stuff which ends up in `custom.config.php`.

**Comment by drybjed** [comment]
If it's just 1 task, I would avoid putting it in a separate task file which is included. The reason is that Ansible v2 evaluates each include dynamically now, and while running the playbook normally doesn't bring much overhead, using tasks makes Ansible slow down to a crawl when it checks each include dynamically for potential tags.

If the number of tasks is small, say 1-3, just put them in the `tasks/main.yml`, especially when they don't have any conditions.

**Comment by Gomez** [comment]
All proposal added. Please review again.

**Comment by drybjed** [comment]
Looks good to me. Thanks! :-)

**Comment by drybjed** [comment]
At the moment ownCloud test on Travis is broken due to missing package repositories for Ubuntu Precise, so I'm going to merge this.

**Pull request merged**